rpcs3/Emu/Cell/SPUThread.cpp

```diff
@@ -532,83 +547,27 @@ void SPUThread::do_dma_transfer(const spu_mfc_cmd& args, bool from_mfc)
 	}
 	default:
 	{
-		auto vdst = static_cast<__m128i*>(dst);
-		auto vsrc = static_cast<const __m128i*>(src);
-		auto vcnt = size / sizeof(__m128i);
+		auto vdst = static_cast<u64*>(dst);
+		auto vsrc = static_cast<const u64*>(src);
+		auto vcnt = size / sizeof(u64);
 
-		//if (is_get && !from_mfc)
+		__movsq(vdst, vsrc, vcnt);
```

Comment:
Nice

Reply:
@kd-11 @Farseer2 why not just use `memcpy` though? Seems like a pessimisation in general. See https://stackoverflow.com/a/43574756/4114447